CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to PHP Women website
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to [PHP Women website](https://github.com/phpwomen/combell/), which is hosted in the [PHP Women Organization](https://github.com/phpwomen/) on GitHub.
+These are just guidelines, not rules, use your best judgment and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Code of Conduct](#code-of-conduct)
+  * [How the Site is Organized](#how-the-site-is-organized)
+
+[How Can I Contribute?](#how-can-i-contribute)
+   * [Reporting Bugs](#reporting-bugs)
+   * [How To Submit Changes](#how-to-submit-changes)
+      * [Pull Requests](#pull-requests)
+
+## What should I know before I get started?
+
+### Code of Conduct
+
+This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code.
+Please report unacceptable behavior to [abuse@phpwomen](mailto:abuse@phpwomen.org).
+
+### How the Site is Organized
+
+We have two nearly identical websites [dev.phpwomen.org](http://dev.phpwomen.org) and [phpwomen.org](http://phpwomen.org).
+
+On GitHub there are two protected branches, `master` which syncs to `dev.phpwomen.org` and `production` which syns with `phpwomen.org`.  This means that we can make changes to the site without affecting production, until we are ready to update the production web site.
+
+## How Can I Contribute?
+
+### Reporting Bugs
```

Comment:
Website bugs must be reported in the Issues section of this project, here on Github.